### ISSUE TITLE
Documentation: Slider control doesn't provide visible textual information of the slider position/value.

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4654]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.3.3 Sensory Characteristics

**Test Environment:**
- Test URL: https://clarity.design/documentation/timeline
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. Open the test URL and try to use the slider controls.

**Expected result:**
The position or value of the slider should be communicated in more than one way. Like slider visual position and textual representation of the position/slider value. 

**Actual result:**
User has to rely only on slider position to understand the set value i.e. to understand the slider position value, user has to rely on position at which slider is placed on bar. So an additional example needs to be added on the page which will have textual representation of the slider position/value.
